### PR TITLE
fix: Delete Space before Commas to format date

### LINKED ISSUE #5122: Date Format: Delete Space before Commas everywhere
In a number of places there is a space in front of a comma, which is against standards in US and British English. Therefore please change this to **no** space before a comma everywhere in the platform and specifically fix it in date displays, e.g. on:
* [ ] dates in front page
* [ ] dashboard of events https://eventyay.com/events
* [ ] all specific event dashboard https://eventyay.com/events/[eventid]
* [ ] admin dashboard of events https://eventyay.com/admin/events
* [ ] users dashboard https://eventyay.com/admin/users/

Screenshot from 2020-09-22 16-04-19

Screenshot from 2020-09-22 16-03-51

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #5122
scrnli_9_23_2020_10-44-35 AM
scrnli_9_23_2020_10-43-46 AM
scrnli_9_23_2020_10-42-14 AM

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by mariobehling — APPROVED** [review]

**Review by Raunaksingh100 — DISMISSED** [review]

**Comment by mariobehling** [comment]
Actually, this bug appears mostly in date formats, but the issue is also related to other areas. 

On the event dashboards there is an info box, where there is also a space before commas. Any other place where you see that issue?

* [ ] session types
* [ ] social media

Screenshot from 2020-09-23 11-41-55

**Review by mariobehling — CHANGES_REQUESTED** [review]
Also delete the space before commas elsewhere.

**Comment by maze-runnar** [comment]
scrnli_9_23_2020_5-24-31 PM
scrnli_9_23_2020_5-23-55 PM

**Comment by maze-runnar** [comment]
> This seems to be changed to lowercase now. The first letter should be capital as before in the social media services names.

it's not due to my change, that is a separate issue.

File: app/templates/components/events/view/overview/general-info.hbs [hunk 1]
```diff
@@ -46,9 +46,9 @@
         <td><strong>{{t 'Session types'}}</strong></td>
         <td>
           {{#if this.data.sessionTypes}}
-            {{#each this.data.sessionTypes as |sessionType index|}}
+            {{#each this.data.sessionTypes as |sessionType index|~}}
               {{if (not-eq index 0) ','}} {{sessionType.name}}
-            {{/each}}
+            {{~/each}}
```
Comment (iamareebjamal) [inline app/templates/components/events/view/overview/general-info.hbs:51]: What does this do?

Comment (maze-runnar) [reply app/templates/components/events/view/overview/general-info.hbs:51]: this removes unwanted space after each loop iteration

Comment (maze-runnar) [reply app/templates/components/events/view/overview/general-info.hbs:51]: https://discuss.emberjs.com/t/removing-whitespace-with-angle-bracket-syntax/17094/9

**Review by mariobehling — APPROVED** [review]
Working for me. Thank you!

**Comment by iamareebjamal** [comment]
A test was failing, I didn't notice

**Pull request merged**